library/pk.c

```diff
@@ -912,24 +912,34 @@ int mbedtls_pk_wrap_as_opaque(mbedtls_pk_context *pk,
 #else /* !MBEDTLS_ECP_LIGHT && !MBEDTLS_RSA_C */
 #if defined(MBEDTLS_ECP_LIGHT)
     if (mbedtls_pk_get_type(pk) == MBEDTLS_PK_ECKEY) {
-        mbedtls_ecp_keypair *ec;
         unsigned char d[MBEDTLS_ECP_MAX_BYTES];
```

Comment:
I agree with the comment, but I have a question. As far as I understood "wrapping as opaque" means that you start from a normal key and then you move it later to the PSA context. However the 1st step is done on the normal side so that key's size should already be fine. Am I wrong?

Btw, as I said, I agree with your comment and I think it's more correct for what we are doing here to use proper PSA defines so I will apply it anyway. Mine is only a curiosity :)

Reply:
No, I don't think you're wrong, it's mostly to make things more obviously right, so that we don't have to reason about where the data comes from to know the size is correct.